include/picongpu/plugins/RandomizedParticleMerger/RandomizedParticleMerger.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2017-2020 Heiko Burau, Xeinia Bastrakova
```

Comment:
@KseniaBastrakova Why did you remove your name?

Reply:
Or more precisely, why did you revert this code?

Reply:
@PrometheusPi as a result, I didn't change a lot I this file, so decided to remove my name from the header